LTHPasscodeViewController/LTHPasscodeViewController.m

```diff
@@ -1093,6 +1096,87 @@ - (void)updateViewConstraints {
                                     multiplier: 1.0f
                                       constant: 0.0f];
         [self.view addConstraints:@[overlayViewLeftConstraint, overlayViewY, overlayViewHeight, overlayViewWidth]];
+        
+        NSLayoutConstraint *topBorderWidth =
+        [NSLayoutConstraint constraintWithItem: _topBorder
```

Comment:
While you can use the old constraintWithItem method, but it is much easier to add constraints using layout anchor: https://developer.apple.com/documentation/uikit/nslayoutanchor?language=objc

like: _topBorder.widthAnchor constraints....

Reply:
updated